addons/vehicles/CfgAmmo.hpp

```diff
@@ -0,0 +1,15 @@
+#define CLASSES_AMMO \
+    "SWS_M_120mm_cannon_ATGM"
+
+
+class Components;
```

Comment:
Since these three are duplicated in both CfgVehicles and CfgAmmo, move them from both to the config.cpp file instead.